FZJDDFCH/iocBoot/iocFZJDDFCH-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,36 @@
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(FZJDDFERMCHOP)/data"
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+## For recsim:
+$(IFRECSIM) drvAsynSerialPortConfigure("L0", "$(PORT=NUL)", 0, 1, 0, 0)
+
+# For dev sim devices
+$(IFDEVSIM) drvAsynIPPortConfigure("L0", "localhost:$(EMULATOR_PORT=)")
```

Comment:
No port set when launched in DEVSIM mode outside of the test framework. Typically, as per the template, we set the default value to 57677

Reply:
Default value added.  At the time of writing the code, there was some confusion over whether or not to specify the port number.  We did not think to look at the template.